### PR TITLE
Feature/complete dcn metadata

### PR DESCRIPTION
**Pull request by zer0stars**
# Proposed Changes

### Impacted Routes
No new routes

### Caveats
The image service still needs to be implemented for DCN.

### CODE REVIEW

File: internal/controllers/nft_controller.go [hunk 1]
```diff
@@ -167,15 +167,36 @@ func (nc *NFTController) GetDcnNFTMetadata(c *fiber.Ctx) error {
 			TraitType: "Creation Date", Value: strconv.FormatInt(dcn.NFTNodeBlockCreateTime.Time.Unix(), 10),
 		})
 	}
+	if dcn.NFTNodeBlockCreateTime.Valid {
+		attrs = append(attrs, NFTAttribute{
+			TraitType: "Registration Date", Value: strconv.FormatInt(dcn.NFTNodeBlockCreateTime.Time.Unix(), 10),
```
Comment (JamesReate) [inline internal/controllers/nft_controller.go:172]: Will this be different than Creation Date? or eventually replace creation date

**Review by JamesReate — APPROVED** [review]
lgtm
thanks for updating tests

**Comment by zer0stars** [comment]
A DCN can expire, and be re-registered by someone else. For now, they are identical.

**Comment by elffjs** [comment]
Can I merge this?

**Pull request merged**